config/puma.rb

```diff
@@ -29,17 +30,35 @@
 # Allow puma to be restarted by `rails restart` command.
 plugin :tmp_restart
 
-# SSL in development (as DFE Signin redirects to https://localhost:3000 but
-# just listen on port 3000 elsewhere
 listen_port = ENV.fetch("PORT") { 3000 }
 
-if Rails.env.development?
+if env == "development"
```

Comment:
I think I prefer `if Rails.env.development?`

Reply:
We can't use `Rails` because we call `puma` directly in the procfile, it's weird...